doc/building/PrecompileVendorPrimitives.rst

```diff
@@ -286,16 +282,16 @@ Selectable Options for the Bash Scripts:
 
   Compile options:
   
-  .. code-block:: raw
+  .. code-block:: none
 	
-        --vhdl93           Compile selected libraries with VHDL-93 (default).
-        --vhdl2008         Compile selected libraries with VHDL-2008.
+     --vhdl93              Compile selected libraries with VHDL-93 (default).
```

Comment:
These options have been indented, because there is no short option. Do you think it looks better with or without the indentation compared to the other code blocks?

Reply:
Then I would prefer the last version with short option after long option :).

Reply:
Done [here](https://github.com/ghdl/ghdl/pull/591/commits/f97dd315cb642534a40b516cd4e54c07b64e9621)